x-pack/plugins/security_solution/server/lib/detection_engine/rules/types.ts

```diff
@@ -181,13 +190,18 @@ export interface CreateRulesOptions {
   immutable: Immutable;
   index: IndexOrUndefined;
   interval: Interval;
+  license: LicenseOrUndefined;
   maxSignals: MaxSignals;
   riskScore: RiskScore;
+  riskScoreMapping: RiskScoreMappingOrUndefined;
```

Comment:
riskScoreMapping should not be `OrUndefined`

Reply:
Thanks for pairing and going over this with me -- changed it for `UpdateRulesOptions` as well! 🙂